src/core/hle/service/hid/controllers/console_sixaxis.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 yuzu emulator team
```

Comment:
ditto with the ones below
```suggestion
// Copyright 2021 yuzu Emulator Project
```